wee_slack.py

```diff
@@ -1062,6 +1099,14 @@ def __init__(self, eventrouter, token, websocket_url, team_info, nick, myidentif
         for c in self.channels.keys():
             channels[c].set_related_server(self)
             channels[c].check_should_open()
+
+        # Build mapper between subteam handle and their identifiers 
+        self.subteam_handles_to_id = {}
+        for s in self.subteams.keys():
+            subteam = self.subteams.get(s, None)
+            if subteam:
+                self.subteam_handles_to_id[subteam.handle] = subteam.identifier
```

Comment:
This should be a method, like `get_username_map`, in case `self.subteams` changes.